docs/source/tutorials/datalab_advanced.ipynb

```diff
@@ -0,0 +1,1241 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Datalab: Advanced workflows to audit your data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`Datalab` helps you identify various issues in your machine learning datasets that may negatively impact the performance of your machine learning model if not addressed. By default, `Datalab` can help you identify noisy labels, outliers, (near) duplicates, and other types of problems that commonly occur in real-world data.\n",
+    "\n",
+    "`Datalab` performs these checks by utilizing the (probabilistic) predictions from *any* ML model that has already been trained or its learned representations of the data. Underneath the hood, this class calls all the appropriate cleanlab methods for your dataset and provided model outputs, in order to best audit the data and alert you of important issues. This makes it easy to apply many functionalities of this library all within a single line of code. \n",
+    "\n",
+    "This tutorial will demonstrate some advanced functionalities of `Datalab` including:\n",
+    "\n",
+    "- Incremental issue search\n",
+    "- Specifying nondefault arguments to issue checks\n",
+    "- Adding a custom IssueManager\n",
+    "- Save and load Datalab objects\n",
```

Comment:
@huiwengoh Can we move save/load section before the custom IssueManager section. The custom section should always remain the final one in this notebook as it is the most advanced part (and not key functionality, unlike save/load)